src/Http/Exceptions/Handler.php

```diff
@@ -2,18 +2,20 @@
 namespace EscolaLms\Files\Http\Exceptions;
 
 use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\Response;
 use Throwable;
 
 class Handler extends \Illuminate\Foundation\Exceptions\Handler
 {
     /**
      * Render an exception into an HTTP response.
      *
-     * @param  \Illuminate\Http\Request  $request
-     * @param  \Throwable  $e
-     * @return \Symfony\Component\HttpFoundation\Response
+     * @param Request $request
+     * @param Throwable $exception
+     * @return Response
      *
-     * @throws \Throwable
+     * @throws Throwable
      */
     public function render($request, Throwable $exception)
```

Comment:
In such situations, try to use a strategy.

Reply:
IMHO this file can be removed `src/Http/Exceptions/Handler.php` 

Isn't is enought just to `throw CustomException` ? @HerbertIV

Reply:
Yes CustomException is fine, but it's worth sending the error status and content for that Exception when raising the exception.

Reply:
Please check my Renderable{} interface solution.